akka-actor-typed-tests/src/test/java/akka/actor/typed/javadsl/ActorCompile.java

```diff
@@ -37,11 +37,21 @@ public MyMsgB(String greeting) {
   Behavior<MyMsg> actor2 = Behaviors.receive((ctx, msg) -> unhandled());
   Behavior<MyMsg> actor4 = empty();
   Behavior<MyMsg> actor5 = ignore();
-  Behavior<MyMsg> actor6 = tap(MyMsg.class, (ctx, signal) -> {}, (ctx, msg) -> {}, actor5);
+  Behavior<MyMsg> actor6 = intercept(new BehaviorInterceptor<MyMsg, MyMsg>() {
+    @Override
+    public Behavior<MyMsg> aroundReceive(ActorContext<MyMsg> ctx, MyMsg msg, ReceiveTarget<MyMsg> target) {
+      return target.apply(msg);
```

Comment:
I guess `apply` has been accepted in java land also? Otherwise we could call them `receive` and `receiveSignal`?

Reply:
It's the name on the method in `java.util.Function` at least, so I think that is fine.